org.mwc.debrief.pepys/src/main/java/org/mwc/debrief/pepys/ImportDatabase.java

```diff
@@ -49,17 +56,38 @@ public Layers getLayers() {
 		};
 
 		final Shell shell = new Shell(PlatformUI.getWorkbench().getDisplay());
+		final DatabaseConfiguration databaseConfiguration = new DatabaseConfiguration();
 		try {
-			new SqliteDatabaseConnection().createInstance();
-			// new PostgresDatabaseConnection().createInstance();
-		} catch (final PropertyVetoException e) {
+			DatabaseConnection.loadDatabaseConfiguration(databaseConfiguration);
+
```

Comment:
the details of the next block look a little untidy, with unused return parameters from the `createInstance()` calls. Let me see if I can improve it.

Reply:
Nope, failed: https://github.com/debrief/debrief/pull/4830